.github/workflows/ci.yml

```diff
@@ -155,9 +155,10 @@ jobs:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/setup-go-for-project
       - shell: bash
-        run: scripts/mock.gen.sh
-      - shell: bash
-        run: .github/workflows/check-clean-branch.sh
+        run: |
+          grep -lr -E '^// Code generated by MockGen\. DO NOT EDIT\.$' . | xargs -r rm
+          go generate -run "go.uber.org/mock/mockgen" ./...
+          git diff --exit-code
```

Comment:
Note: we can revert to use `.github/workflows/check-clean-branch.sh` instead of `git diff --exit-code` but really I think `git diff --exit-code` does it all and we shouldn't need to call an extra script